pkg/ebpf/processor_funcs.go

```diff
@@ -378,3 +379,26 @@ func (t *Tracee) normalizeEventArgTime(event *trace.Event, argName string) error
 	}
 	return nil
 }
+
+// addBinaryInformation enrich the event with processes information.
+// This is done using the process tree of Tracee
+func (t *Tracee) addBinaryInformation(event *trace.Event) error {
+	currentProcess, found := t.processTree.GetProcessByHash(event.ProcessEntityId)
+	if !found {
+		logger.Debugw(
+			"error enriching event executable info",
+			"pid", event.HostProcessID,
+			"tid", event.HostThreadID,
+			"event name", event.EventName,
+			"timestamp", event.Timestamp,
+			"error", "not found in process tree",
+		)
+		return nil
+	}
+	executable := currentProcess.GetExecutable()
+
+	if executable != nil {
```

Comment:
executable wont ever be nil, its initialized together with the node.

Reply:
But isn't it a best practice to check anyways?
Its not important, so if you want to fix it for me you can (I won't be available until Sunday)